src/recipe.rst

```diff
@@ -19,6 +19,29 @@ Your final recipe should have no comments and follow the order in the example.
 *If there are details you are not sure about please open a pull request. The conda-forge team will be happy to answer your questions.*
 
 
+In case you build your first recipe using conda-forge, the we provide here a step-by-step instruction and checklist that might help you with a sucessfully build
```

Comment:
* `you build` -> `you are building`
* `, the we` -> `we have provided` or similar.
* Please remember a full stop (period).